assets/messages.yml

```
@@ -174,6 +174,25 @@ messages:
 
         ~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
 
+        %quick_links%
+      fillname: []
+    - project:
+        - mcpe
+      name: Attach world file (Mobile)
```

```suggestion
      name: Attach world file
```
Reading the thing, this is not only for mobile, it rather seems to be a general MCPE message.